prometheus/value.go

```diff
@@ -215,25 +214,22 @@ func populateMetric(
 // This function is only needed for custom Metric implementations. See MetricVec
 // example.
 func MakeLabelPairs(desc *Desc, labelValues []string) []*dto.LabelPair {
```

Comment:
I think there's an opportunity to move this to `[]fastdto.LabelPair` and pay the price of allocs on Write 🤔 (on scrape)